CoverageExt/CodeRendering/CodeCoverageFactory.cs

```diff
@@ -25,7 +25,7 @@ public sealed class CodeCoverageFactory : IWpfTextViewCreationListener
         /// </summary>
         [Export(typeof(AdornmentLayerDefinition))]
         [Name("CodeCoverage")]
-        [Order(Before = PredefinedAdornmentLayers.BraceCompletion)]
+        [Order(After = PredefinedAdornmentLayers.Selection, Before = PredefinedAdornmentLayers.Text)]
```

Comment:
We should toy with these. Selection/Text is actually too late. Using Before=Selection (without specifying an 'After') might do the trick though. Can you confirm this works?